lib/restAPIs/library.js

```diff
@@ -0,0 +1,105 @@
+//const assert = require('assert');
+
+//const assertParam = require('../util/assertParam');
+const RestUtil = require('../util/RestUtil');
+const buildCommonQuery = require('../util/buildCommonQueryParams');
+
+module.exports = class RestTopic extends RestUtil {
+  /**
+   * Get base endpoint for this API category
+   */
+  _getBase() {
+    return 'library';
+  }
+
+  /**
+   * Return object representing the library for the authenticated user.
+   * @param {array[String]} outputFlag - Filtered output. DEAFULT = include everything, FILTER_TOC = filter out cahpters and pages.
```

Comment:
Primitive type for string: `@param {array[string]}`

Reply:
In case you're wondering, I plan on changing all types to primitives for JSDoc comments #26 - but going forward it's good to start doing this in new additions too.

Reply:
fixed.